doc/releases/changelog-dev.md

```diff
@@ -295,6 +295,29 @@
 
 <h3>Deprecations</h3>
 
+* In-place inversion is now deprecated. This includes `op.inv()` and `op.inverse=value`. Please
+  use `qml.adjoint` instead. Support for these methods will remain till v0.28.
```

Comment:
Or comparable, as per the previous comment.
```suggestion
  use `qml.adjoint` or exponentiating the operator to the `-1`th power instead. Support for these methods will remain till v0.28.
```

Reply:
Actually adding a `qml.pow` constructor in PR #3029 .  That will be easier to reference here than `op**-1`.  Should be a quick PR, then we can update this after that PR is merged.

Reply:
Can we merge this as suggested and #3029 will update the wording? That way we're on the safe side.

Reply:
I can do that.  If this PR is merged first, I'll update the wording of the warning in #3029.